### ISSUE TITLE
[BUG][A11Y][Select] Accessibility Issue

### ISSUE THREAD

**Issue by Hubery-Shen** [SAP/ui5-webcomponents#2190]
**Dear colleagues,**
I am from SuccessFactors Web Experience Shanghai team.

Lose many a11y attributes: 
1. aria-roledescription
2. aria-disabled
3. aria-readonly
4. aria-expanded
5. aria-controls
6. aria-activedescendant
7. aria-selected
8. aria-setsize
9. aria-posinset

<img width="1099" alt="BUG  select-3" src="https://user-images.githubusercontent.com/63704163/92445932-ab2fd700-f1e7-11ea-80b1-f9618c005a7e.png">
<img width="1065" alt="BUG  Select-1" src="https://user-images.githubusercontent.com/63704163/92444180-175d0b80-f1e5-11ea-82f2-dc4084bdc1ec.png">

**Comment by fifoosid** [comment]
Hello @Hubery-Shen 

I am merging a PR, fixing this issue. However, after a sync with our ACC experts we reached the following conclusion([More info here](https://github.com/SAP/ui5-webcomponents/pull/2204#discussion_r490744725)):

```aria-roledescription``` - added
```aria-disabled``` - already implemented
```aria-readonly``` - not needed, as the select doesn't have readonly property
```aria-expanded``` - added
```aria-controls``` - not needed, because we have a aria-haspopup=listbox
```aria-activedescendant``` -  could not be implemented since the items are in a different DOM
```aria-selected``` - added
```aria-setsize``` - added
```aria-posinset``` - added

BR,
Filip

**Comment by Milo-Shen** [comment]
Hi @fifoosid 

Thanks for your support
